LibExeObjStep.installRaw is outdated and not tested by the CI

When invoking `installRaw`, I get the following error:

```bash
C:\...\build\emit_raw.zig:95:18: error: expected 4 arguments, found 3
        sort.sort(*BinaryElfSegment, self.segments.span(), segmentSortCompare);
```

It seems that `emit_raw.zig` uses the old sort implementation and hasn't been modernized, and that this wasn't caught due to it being missing in the Zig test suite.

--- a/lib/std/build.zig
+++ b/lib/std/build.zig
@@ -2558,3 +2558,10 @@ pub const InstalledFile = struct {
     dir: InstallDir,
     path: []const u8,
 };
+
+test "" {
+    // The only purpose of this test is to get all these untested functions
+    // to be referenced to avoid regression so it is okay to skip some targets.
+    if (comptime std.Target.current.cpu.arch.ptrBitWidth() == 64)
+        std.meta.refAllDecls(@This());
+}

--- a/lib/std/build/emit_raw.zig
+++ b/lib/std/build/emit_raw.zig
@@ -215,3 +215,7 @@ pub const InstallRawStep = struct {
         try emitRaw(builder.allocator, full_src_path, full_dest_path);
     }
 };
+
+test "" {
+    std.meta.refAllDecls(InstallRawStep);
+}

